2020q3/puppet.md

```diff
@@ -0,0 +1,34 @@
+## Puppet
+
+Link:	 [PuppetLab's FreeBSD slack channel](https://puppetcommunity.slack.com/messages/C6CK0UGB1/)
```

Comment:
Would change this to "Puppet's FreeBSD Slack channel" since Puppet Labs is no longer used as the company name.

Reply:
Done!